api/_globals/crypto.json

```diff
@@ -0,0 +1,108 @@
+{
+  "api": {
+    "crypto": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/crypto_property",
+        "spec_url": "https://w3c.github.io/webcrypto/#dom-windoworworkerglobalscope-crypto",
+        "support": {
+          "chrome": {
+            "version_added": "37"
+          },
+          "chrome_android": {
+            "version_added": "37"
+          },
+          "deno": {
+            "version_added": "1.0"
+          },
+          "edge": {
+            "version_added": "12"
+          },
+          "firefox": {
+            "version_added": "1"
+          },
+          "firefox_android": {
+            "version_added": "4"
+          },
+          "ie": {
+            "version_added": "11",
+            "prefix": "ms"
+          },
+          "opera": {
+            "version_added": "24"
+          },
+          "opera_android": {
+            "version_added": "24"
+          },
+          "safari": {
+            "version_added": "5"
+          },
+          "safari_ios": {
+            "version_added": "5"
+          },
+          "samsunginternet_android": {
+            "version_added": "3.0"
+          },
+          "webview_android": {
+            "version_added": "37"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "worker_support": {
+        "__compat": {
+          "description": "Available in workers",
+          "support": {
+            "chrome": {
+              "version_added": "37"
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "deno": {
+              "version_added": null
```

Comment:
@lucacasonato can you help me find the right version for `self.crypto` support in Deno?

Reply:
https://deno.com/blog/v1.11#:~:text=We%20have%20supported%20crypto.getRandomValues()%20since%20Deno%201.0 - According to this blog post crypto in some form has been supported since 1.0

Reply:
Yeah, self.crypto has been around since 1.0. The `Crypto` interface was only added later if I remember right.

Reply:
Thanks, I'll go with 1.0!